tests/scripts/users/test_migrate_users_roles.py

```diff
@@ -0,0 +1,236 @@
+from collections import Counter
+
+import pytest
+
+import pcapi.core.offers.factories as offers_factories
+import pcapi.core.users.factories as users_factories
+from pcapi.core.users.models import UserRole
+from pcapi.scripts.users.migrate_users_roles import migrate_users_roles
+
+
+@pytest.mark.usefixtures("db_session")
+def test_migrate_beneficiary_with_no_roles():
```

Comment:
FYI, une autre syntaxe existe si tu veux pour un test uniquement c'est
```suggestion
def test_migrate_beneficiary_with_no_roles(db_session):
```

Et puisqu'ici tous les tests en ont besoin tu peux aussi utiliser la syntaxe 

```
pytestmark = pytest.mark.usefixtures("db_session")
```

En haut du fichier :)

Reply:
Historiquement, le décorateur était pour remplacer très facilement le @clean_database. Depuis par mimétisme il est repris (notamment il est pratique sur les classes) mais voilà, c'est plus classique d'injecter des fixtures directement dans la définition de la méthode :)

Reply:
Anyway c'est un fichier temporaire donc ne prend pas ces retours en compte, c'était au cas où